dist/index.html

```diff
@@ -76,10 +66,17 @@ <h3>Customer Bookings</h3>
         </section>
       </section>
     </div>
+    <div class="available-rooms-div hidden">
+      <div class="available-title">
+      <h3>Available Rooms</h3>
+      </div>
+    <section class="available-bookings">
+      <section class="available-rooms-card">
+      </section>
+    </section>
+  </div>
 </main>
   <!-- Do not include the scripts.js or class files here - it is done by the webpack server -->
```

Comment:
Consider taking out this comment explaining how the webpack works to bundle data.